repos/fdbt-site/src/utils/apiUtils/userData.ts

```diff
@@ -386,6 +388,11 @@ export const getReturnTicketJson = (req: NextApiRequestWithSession, res: NextApi
 
     const { service, userFareStages, matchingFareZones } = matchingAttributeInfo;
 
+    let returnService = undefined;
```

Comment:
I think we can remove these lines - 391-395, and then...

Reply:
done